cruise-control/src/integrationTest/java/com/linkedin/kafka/cruisecontrol/DiskFailureIntegrationTest.java

```diff
@@ -55,6 +61,21 @@ public class DiskFailureIntegrationTest extends CruiseControlIntegrationTestHarn
   private static final Logger LOG = LoggerFactory.getLogger(DiskFailureIntegrationTest.class);
   private final Configuration _gsonJsonConfig = KafkaCruiseControlIntegrationTestUtils.createJsonMappingConfig();
   private final List<Entry<File, File>> _brokerLogDirs = new ArrayList<>(KAFKA_CLUSTER_SIZE);
+  private final Boolean _vertxEnabled;
+
+  public DiskFailureIntegrationTest(Boolean vertxEnabled) {
+    this._vertxEnabled = vertxEnabled;
+  }
+
+  /**
+   * Sets different parameters for test runs.
+   * @return Parameters for the test runs.
+   */
+  @Parameterized.Parameters
+  public static Collection<Boolean> data() {
+    Boolean[] data = {false, false};
```

Comment:
Shouldn't this be true, false?

Reply:
Yes, this is a stupid mistake. I will correct it. Thank you for pointing it out.